internal/clientsholder/clientsholder.go

```diff
@@ -37,19 +37,21 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 	apiextv1fake "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
 	k8sFakeClient "k8s.io/client-go/kubernetes/fake"
+	networkingv1 "k8s.io/client-go/kubernetes/typed/networking/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 type ClientsHolder struct {
-	RestConfig    *rest.Config
-	DynamicClient dynamic.Interface
-	APIExtClient  apiextv1.Interface
-	OlmClient     olmClient.Interface
-	OcpClient     clientconfigv1.ConfigV1Interface
-	K8sClient     kubernetes.Interface
-	MachineCfg    ocpMachine.Interface
-	ready         bool
+	RestConfig          *rest.Config
+	DynamicClient       dynamic.Interface
+	APIExtClient        apiextv1.Interface
+	OlmClient           olmClient.Interface
+	OcpClient           clientconfigv1.ConfigV1Interface
+	K8sClient           kubernetes.Interface
+	K8sNetworkingClient networkingv1.NetworkingV1Interface
```

Comment:
This client can be gotten from the K8sClient (`clientsHolder.K8sClient.NetworkingV1()`).

Reply:
I was trying to make it consistent with how we use the normal `K8sClient` and passing around an interface type.  I tried getting a mock client up and mocking out all of the function calls to get the network policies but didn't have any luck.   The `networkingv1` import doesn't have the same fake client support it seems.